functions/Invoke-DbaDbMirroring.ps1

```diff
@@ -317,18 +320,20 @@ function Invoke-DbaDbMirroring {
                 $serviceAccounts = $source.ServiceAccount, $dest.ServiceAccount, $witserver.ServiceAccount | Select-Object -Unique
 
                 foreach ($account in $serviceAccounts) {
-                    if ($Pscmdlet.ShouldProcess("primary, mirror and witness (if specified)", "Creating login $account and granting CONNECT ON ENDPOINT")) {
-                        $null = New-DbaLogin -SqlInstance $source -Login $account -WarningAction SilentlyContinue
-                        $null = New-DbaLogin -SqlInstance $dest -Login $account -WarningAction SilentlyContinue
-                        try {
-                            $null = $source.Query("GRANT CONNECT ON ENDPOINT::$primaryendpoint TO [$account]")
-                            $null = $dest.Query("GRANT CONNECT ON ENDPOINT::$currentmirrorendpoint TO [$account]")
-                            if ($witserver) {
-                                $null = New-DbaLogin -SqlInstance $witserver -Login $account -WarningAction SilentlyContinue
-                                $witserver.Query("GRANT CONNECT ON ENDPOINT::$witnessendpoint TO [$account]")
+                    if ($account) {
```

Comment:
How can $account ever be $false?

Reply:
if it's blank, like with docker containers somehow

![image](https://user-images.githubusercontent.com/8278033/123311040-fcb31380-d526-11eb-838c-ed911a1a1d87.png)

Reply:
Ok, I see.